grakn-engine/src/main/java/ai/grakn/engine/backgroundtasks/InMemoryTaskManager.java

```diff
@@ -163,6 +183,7 @@ private Runnable exceptionCatcher(String id, BackgroundTask task) {
                 stateUpdateLock.unlock();
             }
             catch (Throwable t) {
+                LOG.error(getFullStackTrace(t));
```

Comment:
I would say these error logs/stack traces need context. Just printing stack traces can be a bit confusing. When did the failure occur for example ? Even just knowing the task it failed during would be nice.